mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdToImapUidDAO.java

```diff
@@ -148,7 +149,24 @@ private PreparedStatement prepareInsert(Session session) {
         if (cassandraConfiguration.isMessageWriteStrongConsistency()) {
             return session.prepare(insert.ifNotExists());
```

Comment:
Should we move `Insert insert = ... ` into the `if` block.

Reply:
Yes if we do not want to have yet-another method.

Given that the duplicated code is trivial, duplication here IMO is harmless.